GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Remotes.cs

```diff
@@ -287,6 +288,13 @@ public bool Prune()
                 return DoPrune();
             }
 
+            public void OpenRemoteUrlInBrowser()
+            {
+                Process.Start(_remote.FetchUrl);
```

Comment:
We still need to check here whether we _can_ open the URL.
The fact that the menu is not visible is orthogonal, and there is no obvious connection between these two. It is not difficult to imagine a situation where this method gets invoked programmatically without any safe guards in place.

```cs
            public void OpenRemoteUrlInBrowser()
            {
				if (!IsRemoteUrlUsingHttp)
				{
					return;
				}

                Process.Start(_remote.FetchUrl);
```

Reply:
Alrighty. will update it on weekends.